data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/JsonTypeDefinitionBeaconRestApi.java

```diff
@@ -165,145 +165,154 @@ private static RestApi create(
-        .endpoint(new GetPeers(dataProvider))
-        .endpoint(new GetPeerCount(dataProvider))
-        .endpoint(new GetPeerById(dataProvider))
-        .endpoint(new GetSyncing(dataProvider))
-        .endpoint(new GetVersion())
-        // Validator Handlers
-        .endpoint(new PostAttesterDuties(dataProvider))
-        .endpoint(new GetProposerDuties(dataProvider))
-        .endpoint(
-            new tech.pegasys.teku.beaconrestapi.handlers.v1.validator.GetNewBlock(
-                dataProvider, schemaCache))
-        .endpoint(new GetNewBlock(dataProvider, spec, schemaCache))
-        .endpoint(new GetNewBlindedBlock(dataProvider, spec, schemaCache))
-        .endpoint(new GetAttestationData(dataProvider))
-        .endpoint(new GetAggregateAttestation(dataProvider, spec))
-        .endpoint(new PostAggregateAndProofs(dataProvider, spec.getGenesisSchemaDefinitions()))
-        .endpoint(new PostSubscribeToBeaconCommitteeSubnet(dataProvider))
-        .endpoint(new PostSyncDuties(dataProvider))
-        .endpoint(new GetSyncCommitteeContribution(dataProvider, schemaCache))
-        .endpoint(new PostSyncCommitteeSubscriptions(dataProvider))
-        .endpoint(new PostContributionAndProofs(dataProvider, schemaCache))
-        .endpoint(new PostPrepareBeaconProposer(dataProvider))
-        .endpoint(new PostRegisterValidator(dataProvider))
-        // Config Handlers
-        .endpoint(
-            new GetDepositContract(
-                config.getEth1DepositContractAddress(), dataProvider.getConfigProvider()))
-        .endpoint(new GetForkSchedule(dataProvider))
-        .endpoint(new GetSpec(dataProvider))
-        // Debug Handlers
-        .endpoint(new GetChainHeadsV1(dataProvider))
-        .endpoint(new GetChainHeadsV2(dataProvider))
-        .endpoint(
-            new tech.pegasys.teku.beaconrestapi.handlers.v1.debug.GetState(
-                dataProvider, spec, schemaCache))
-        .endpoint(new GetState(dataProvider, schemaCache))
-        .endpoint(new GetForkChoice(dataProvider))
-        // Teku Specific Handlers
-        .endpoint(new PutLogLevel())
-        .endpoint(new GetStateByBlockRoot(dataProvider, spec))
-        .endpoint(new Liveness(dataProvider))
-        .endpoint(new Readiness(dataProvider, executionClientDataProvider))
-        .endpoint(new GetAllBlocksAtSlot(dataProvider, schemaCache))
-        .endpoint(new GetPeersScore(dataProvider))
-        .endpoint(new GetProposersData(dataProvider))
-        .endpoint(new GetDeposits(eth1DataProvider))
-        .endpoint(new GetEth1Data(dataProvider, eth1DataProvider))
-        .endpoint(new GetEth1DataCache(eth1DataProvider))
-        .endpoint(new GetEth1VotingSummary(dataProvider, eth1DataProvider))
-        .endpoint(new GetDepositSnapshot(eth1DataProvider))
-        .endpoint(new GetGlobalValidatorInclusion(dataProvider))
-        .endpoint(new GetValidatorInclusion(dataProvider))
-        .build();
+    RestApiBuilder builder =
```

Comment:
Looks like a big diff, but it is simply moving the two light client endpoints to the end of function to evaluate whether the flag is set.